fahrschein-http-spring/src/main/java/org/zalando/fahrschein/http/spring/HeadersAdapter.java

```diff
@@ -18,7 +19,7 @@ class HeadersAdapter implements Headers {
 
     @Override
     public List<String> get(String headerName) {
-        return headers.get(headerName);
+        return Optional.ofNullable(headers.get(headerName)).orElse(Collections.emptyList());
```

Comment:
Found a surprisingly differing behaviour in the get(headerName) implementations: Spring was returning null, but in our basic HeadersImpl (reference impl?) we were specifying that get on an unknown header would return an emptyList.